config/standard/resources.json

```diff
@@ -1,26 +1,25 @@
 {
    "area": "icon-places-CHW-area@2x.png",
-   "branch": "icon-places-clinic@2x.png",
+   "branch": "icon-places-clinic@2x.svg",
    "child": "icon-people-child@2x.png",
    "children": "icon-people-children@2x.png",
-   "clinic": "icon-places-clinic@2x.png",
-   "family": "icon-people-family@2x.png",
+   "clinic": "icon-places-clinic@2x.svg",
+   "family": "icon-people-family@2x.svg",
    "follow-up": "icon-followup-general@2x.png",
    "household": "icon-places-household@2x.png",
    "immunization": "icon-healthcare-immunization@2x.png",
    "infant": "icon-people-baby@2x.png",
    "mother-child": "icon-people-woman-baby@2x.png",
-   "new-person": "icon-people-person-general@2x.png",
+   "new-person": "icon-people-person-general@2x.svg",
    "nurse": "icon-people-nurse@2x.png",
    "off": "icon-messages-off@2x.png",
    "on": "icon-messages-on@2x.png",
-   "person": "icon-people-person-general@2x.png",
+   "person": "icon-people-person-general@2x.svg",
```

Comment:
I think we should remove the `2x` suffix from these file names - it doesn't make sense with svgs as they scale to any size.

Reply:
Ok